Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -33,14 +33,14 @@ public sealed class NetworkIdentity : MonoBehaviour
 
         // properties
         public bool isClient { get; internal set; }
-        public bool isServer => m_IsServer && NetworkServer.active; // dont return true if server stopped.
+        public bool isServer { get => m_IsServer && NetworkServer.active && netId != 0; internal set => m_IsServer = value; } // dont return true if server stopped.
```

Comment:
at least put this into multiple lines please. this is very hard to read.

Reply:
Done